src/main/java/com/cloudbees/jenkins/GitHubCause.java

```diff
@@ -0,0 +1,41 @@
+package com.cloudbees.jenkins;
+
+import hudson.triggers.SCMTrigger.SCMTriggerCause;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * UI object that says a build is started by GitHub post-commit hook.
+ *
+ * @author Marcus Chan
+ */
+public class GitHubCause extends SCMTriggerCause {
+
+    private String eventType;
```

Comment:
I would explicitly mark fields and methods as <code>@CheckForNull</code>. There's inconsistency in the code: you default the value to empty string, but check against null in <code>getShortDescription</code>